docs/DeveloperGuide.md

```diff
@@ -116,16 +116,44 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `MATHTUTORING` and the **Actor** is the `Tutor`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Delete a student**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  Tutor requests to list students
+2.  MATHTUTORING shows a list of students
+3.  Tutor requests to delete a specific student in the list
+4.  MATHTUTORING deletes the student
```

Comment:
Should we add a period at the end of each line? I think it's optional (AB3 doesn't use periods but the textbook does) but perhaps if you want to, can make it uniform with the lines below??? :DDDDD

Reply:
Done